### ISSUE TITLE
Editor: Can't edit link URL right after pasting URL into text

### ISSUE THREAD

**Issue by mcsf** [Automattic/wp-calypso#21577]
#### Steps to reproduce
1. Copy a URL onto the clipboard, e.g. `https://wordpress.org`.
2. Start editing a post with some text.
3. Select a fragment of text.
4. Press `Mod+V` to paste. A tooltip highlighting the new link should appear.
5. Click the pencil icon on the tooltip. The link-editing modal should appear.

#### What I expected
The first field of the modal, URL, should contain the pasted URL.

#### What happened instead
The field is empty.

If the modal is dismissed then opened again, the URL is shown.

#### Browser / OS version
Firefox 58.x (Quantum), latest Chrome and Safari. macOS 10.13.x (High Sierra).

#### Screenshot / Video
![calypso-edit-link](https://user-images.githubusercontent.com/150562/35018011-903c021c-fb17-11e7-9272-ad5547c2826c.gif)

**Comment by lancewillett** [comment]
Confirmed this is still an issue, testing in Safari 11.0.3 on OS X 10.13.3.

**Comment by rachelmcr** [comment]
I tested with macOS 10.13.6 / Chrome 68.0.3440.106 and confirmed this is still an issue. (One exception is if you select the text at the very end of a line, in which case the modal includes the pasted URL.)

However, I also tested in Gutenberg and confirmed the issue does not appear there. The pasted URL always appears in the link editing modal. Given that we only plan to make critical fixes to the Calypso editor, and this is not an issue in Gutenberg, I'm closing this issue for now.
